new-findings.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+GO_SRC_PATH="$(go env GOPATH)/src"
+CURRENT_BRANCH=$(git branch --show-current)
+
+if [[ ${CURRENT_BRANCH} == "master" ]]; then
```

Comment:
Quote wrap `${CURRENT_BRANCH}`.  Also check for non-empty.

Reply:
Done.